escheduler-ui/src/js/module/i18n/locale/zh_CN.js

```diff
@@ -474,5 +474,11 @@ export default {
   'warning of timeout': '超时告警',
   'Next five execution times': '接下来五次执行时间',
   'Execute time': '执行时间',
-  'Complement range': '补数范围'
+  'Complement range': '补数范围',
+  'Complement range': '补数范围',
```

Comment:
`Complement range` repeated 3 times ？

and en_US.js also should be updated!

Reply:
Updated